### PR TITLE
FIX Stratux so it works again!!! a.k.a. added stratux-wifi.sh to update scripts and correct pathing issue

### PR DESCRIPTION
**Pull request by peepsnet**
I added the needed lined to include stratux-wifi.sh  when the update scripts is run

### CODE REVIEW

**Comment by Ergonomicmike** [comment]
And the last two commits prior to this bug fix should be removed, since they don't work. (Per peepsnet in Slack.)

File: image/stratux-wifi.sh [hunk 1]
```diff
@@ -20,7 +23,7 @@ RPI_REV=`cat /proc/cpuinfo | grep 'Revision' | awk '{print $3}' | sed 's/^1000//
 if [ "$RPI_REV" = "a01041" ] || [ "$RPI_REV" = "a21041" ] || [ "$RPI_REV" = "900092" ] || [ "$RPI_REV" = "900093" ] && [ "$EW7811Un" != '' ]; then
  # This is a RPi2B or RPi0 with Edimax USB Wifi dongle.
  DAEMON_CONF=/etc/hostapd/hostapd-edimax.conf
- DAEMON_SBIN=/etc/hostapd/hostapd-edimax
+ DAEMON_SBIN=/usr/sbin//hostapd-edimax
```
Comment (cyoung) [inline image/stratux-wifi.sh]: Double slash.

**Comment by cyoung** [comment]
Removed `update-stratux-v1.1b1-3e663c89c8.sh` and `update-stratux-v1.1b1-de93c79a07.sh`.

File: image/stratux-wifi.sh (hunk 1 above)
Comment (peepsnet) [inline image/stratux-wifi.sh]: Done

**Comment by peepsnet** [comment]
Should be GTG now!!

**Comment by cyoung** [comment]
Thanks!

**Pull request merged**